media-sound/mpg123/files/mpg123-1.32.3-build-with-installed-libs.patch

```diff
@@ -0,0 +1,251 @@
+build: allow building with installed libmpg123 libsyn123 libout123
```

Comment:
Could you add a link to the Gentoo bug and upstream PR at the top of each of the patches?

Reply:
Done!